Error "Cannot find module 'babylon'" when running global build

Running global build via `esri-wab-build` results in error:

`module.js:487
    throw err;
    ^
Error: Cannot find module 'babylon'`

Likely Babylon should be added to package.json as a dependency due to it's use in build.

--- a/package.json
+++ b/package.json
@@ -32,6 +32,7 @@
     "url": "https://github.com/gbochenek/esri-wab-build"
   },
   "dependencies": {
+    "babylon": "^6.17.4",
     "fs-extra": "^3.0.1",
     "husky": "^0.14.1",
     "lint-staged": "^4.0.0",

